### PR TITLE
Clarify email

### PR DESCRIPTION
**Pull request by alizauf**
I added language to flash messages and email so that vendors who follow specific opportunities are prompted to sign up for the newsletter. This was in reference to #1

### CODE REVIEW

File: beacon/blueprints/front.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def browse():
         if signup_for_opp(
             signup_form, opportunity=opportunities, multi=True
         ):
-            flash('Successfully subscribed for updates!', 'alert-success')
+            flash(Markup("Successfully subscribed for updates! <a href='/signup'>Sign up</a> for alerts about future opportunities."), 'alert-success')
```
Comment (ajb) [inline beacon/blueprints/front.py]: Can we use the `url_for` method instead of writing `/signup`?

File: beacon/forms/opportunities.py [hunk 2]
```diff
@@ -225,7 +225,7 @@ class OpportunitySignupForm(Form):
     def post_validate_action(self, opportunity):
         signup_success = signup_for_opp(self, opportunity)
         if signup_success:
-            flash('Successfully subscribed for updates!', 'alert-success')
+            flash(Markup("Successfully subscribed for updates! <a href='/signup'>Sign up</a> for alerts about future opportunities."), 'alert-success')
```
Comment (ajb) [inline beacon/forms/opportunities.py]: Ditto https://github.com/dobtco/beacon/pull/25/files#r58381254

File: beacon/templates/beacon/emails/signup.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 {% if categories|length > 20 %}
 <p>You've signed up to receive notifications about {{ categories|length }} categories. You can view all of them <a href="{{ url_for('front.manage', _external=True) }}">here.</a></p>
-{% else %}
+{% elif categories|length > 0 %}
```
Comment (ajb) [inline beacon/templates/beacon/emails/signup.html]: :+1:

File: test/integration/beacon/test_front.py [hunk 4]
```diff
@@ -290,7 +290,7 @@ def test_signup_for_multiple_opportunities(self):
 
         # should redirect and flash properly
         self.assertEquals(post.status_code, 302)
-        self.assert_flashes('Successfully subscribed for updates!', 'alert-success')
+        self.assert_flashes("Successfully subscribed for updates! <a href='/signup'>Sign up</a> for alerts about future opportunities.", 'alert-success')
```
Comment (ajb) [inline test/integration/beacon/test_front.py]: I think it's probably OK to manually specify the `/signup` path here, since it's a test.

**Comment by ajb** [comment]
@alizauf: I added some comments to this PR for changes that should be made before it gets merged. You can checkout this branch (`clarify-email`), and any commits you push will be appended to this PR. Once you're done, write a comment to let me know, and I'll review it again.

Also, it looks like there's a test (test_signup_for_opportunity) that's failing.

**Comment by alizauf** [comment]
Cool I'll make these changes. Thanks.

**Comment by alizauf** [comment]
I think that test is failing due to the new added link--I'll work on that.

**Comment by alizauf** [comment]
Okay, I believe I fixed all of these. I added Markup to the tests since it is used in the flashes, but I'm not sure if that will cause issues with the tests. Is there a way for me to run the tests locally?

**Comment by ajb** [comment]
https://github.com/dobtco/beacon#testing

**Comment by alizauf** [comment]
Ah, yep, that was right there.

**Comment by ajb** [comment]
Looks great. Merged!

**Pull request merged**